src/v2/ion/i18nTransformer.js

```diff
@@ -61,6 +62,8 @@ const I18N_OVERRIDE_MAPPINGS = {
   'select-authenticator-authenticate.authenticator.okta_verify.signed_nonce': 'oie.okta_verify.signed_nonce.label',
   'select-authenticator-authenticate.authenticator.okta_verify.push': 'oie.okta_verify.push.title',
   'select-authenticator-authenticate.authenticator.okta_verify.totp': 'oie.okta_verify.totp.title',
+  'select-authenticator-authenticate.authenticator.google_authenticator':
+    'oie.authenticator.google_authenticator.label',
```

Comment:
we need one for enroll form 

"enroll-authenticator.google_authenticator. credentials.otp":  "enter code", 

https://github.com/okta/okta-signin-widget/pull/1602/commits/d692d205ffc1447ec68dd2b5a549427e91e83800#diff-5974c7a70bf384765e488ae56c6181fc159c61fcc7e8a6e4a90a8052793519cfR28